apps/server/src/modules/request/domain/notify-request-answer-set.event-handler.ts

```diff
@@ -57,11 +57,16 @@ function getContext(
   request: Request & { requester: Member },
   respondent: Member,
   answer: RequestAnswer['answer'] | null,
+  previousAnswer: RequestAnswer['answer'] | null,
 ): ReturnType<GetNotificationContext<'RequestAnswerSet'>> {
   if (member.id !== request.requester.id || respondent.id === request.requester.id) {
     return null;
   }
 
+  if (answer !== 'positive' && previousAnswer !== 'positive') {
```

Comment:
I think this works, but I struggled to understand it (no action needed)

Reply:
Yes, I don't have a name to use for this condition. Because it is:

- negative answer withdrawn
- new negative answer given

at the same time

Reply:
It describes the domain rules, so I prefer your suggestion. I'll update the code